sabnzbd/notifier.py

```diff
@@ -30,7 +30,7 @@
 import json
 from threading import Thread
 from typing import Optional, Dict
-
+import apprise
```

Comment:
For aesthetics, let's add a newline after `apprise`.

Reply:
Done